### DISCUSSION TITLE
modpack/kubejs/1.20.1/Introduction/Addon/ProbeJS/ProbeJS

### DISCUSSION THREAD

**Discussion by giscus** [PickAID/CrychicDoc#37] [Announcements]
# modpack/kubejs/1.20.1/Introduction/Addon/ProbeJS/ProbeJS

一个包含 Minecraft 开发文档的网站。

https://docs.mihono.cn/zh/modpack/kubejs/1.20.1/Introduction/Addon/ProbeJS/ProbeJS

**Comment by XxttCrystal** [comment]
没有提及：所有Js代码必须在src文件夹下，否则不会提供补全操作。

**Reply by M1hono** [reply]
毕竟1.20.1的7.0.X版本已不再受支持（且已下架），所以不会特地标注，但是提供了7.0.X迁移回6.0.1的方法。
